test/context_manager.spec.ts

```diff
@@ -8,10 +8,12 @@ describe("Test context manager", () => {
   const extensionNameA = "aaa";
   const extensionNameB = "bbb";
 
-  const USER = "koko@sap.com";
-  const ENVIRONMENT = "prod";
-  const LANDSCAPE = "eu10";
-  const TENANTID = "111-a11-111";
+  const user = "koko@sap.com";
```

Comment:
out of curiosity, why did you change it to small letters?

Reply:
because it is used in the test (parameter) and not as const